skactiveml/classifier/_wrapper.py

```diff
@@ -714,3 +717,132 @@ def __getattr__(self, item):
+        during the fit.
+    missing_label : scalar or string or np.nan or None, default=np.nan
+        Value to represent a missing label.
+    cost_matrix : array-like of shape (n_classes, n_classes)
+        Cost matrix with `cost_matrix[i,j]` indicating cost of predicting class
+        `classes[j]` for a sample of class `classes[i]`. Can be only set, if
+        `classes` is not none.
+    random_state : int or RandomState instance or None, default=None
+        Determines random number for 'predict' method. Pass an int for
+        reproducible results across multiple method calls.
+    **kwargs : keyword arguments
+        more possible parameters to customizing your neural network
+        see: https://skorch.readthedocs.io/en/stable/net.html
+
+    References
+    ----------
+    [1] Marian Tietz, Thomas J. Fan, Daniel Nouri, Benjamin Bossan, and
+    skorch Developers. skorch: A scikit-learn compatible neural network
+    library that wraps PyTorch, July 2017.
+    """
+
+    def __init__(
+        self,
+        module,
+        criterion=nn.NLLLoss(),
+        classes=None,
+        missing_label=MISSING_LABEL,
+        cost_matrix=None,
+        random_state=None,
+        **kwargs,
+    ):
+        super(SkorchClassifier, self).__init__(
+            module,
+            criterion,
+            **kwargs,
+        )
+
+        SkactivemlClassifier.__init__(
+            self,
+            classes=classes,
+            missing_label=missing_label,
+            cost_matrix=cost_matrix,
+            random_state=random_state,
+        )
+
+    def fit(self, X, y, **fit_params):
+        """Initialize and fit the module.
+
+        If the module was already initialized, by calling fit, the
+        module will be re-initialized (unless ``warm_start`` is True).
+
+        Parameters
+        ----------
+        X : matrix-like, shape (n_samples, n_features)
+            Training data set, usually complete, i.e. including the labeled and
+            unlabeled samples
+        y : array-like of shape (n_samples, )
+            Labels of the training data set (possibly including unlabeled ones
+            indicated by self.missing_label)
+        fit_params : dict-like
+            Further parameters as input to the 'fit' method of the 'estimator'.
+
+        Returns
+        -------
+        self: SkorchClassifier,
+            The SkorchClassifier is fitted on the training data.
+        """
+
+        # check input parameters
+        self.check_X_dict_ = {
+            "ensure_min_samples": 0,
+            "ensure_min_features": 0,
+            "allow_nd": True,
+            "dtype": None,
+        }
+        X, y, sample_weight = self._validate_data(
+            X=X,
+            y=y,
+            check_X_dict=self.check_X_dict_,
+        )
+
+        is_lbld = is_labeled(y, missing_label=self.missing_label)
+        if np.sum(is_lbld) == 0:
+            raise ValueError("There is no labeled data.")
```

Comment:
I think we should not raise an error but initialize the network if this is not done so far.